app/containers/Registration/OrganisationListContainer.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {graphql, createFragmentContainer} from 'react-relay';
+import {Dropdown, DropdownButton} from 'react-bootstrap';
+
+const OrganisationListContainer = props => {
+  const {query} = props;
+
+  // Const query = graphql`
+  //   query OrganisationListContainerQuery {
+  //     query {
+  //       ...OrganisationListContainer_query
+  //     }
+  //   }
+  // `;
```

Comment:
Remove commented code

Reply:
This whole file is not used. I will remove it.

Reply:
Removed this file.